input.txt

```diff
@@ -0,0 +1 @@
+a
```

Comment:
Yeah, it's sort of leftover from manual testing. It was used as an input. If you enter:
```
$ sbt "run exe main.morganey" < input.txt
```

It will print `string: c`, which is proof concept of Morganey interaction with outside world. Maybe those `input.txt` and `main.morganey` should be part of documentation and relate #106. Let me remove them...